composure.sh

```diff
@@ -175,15 +175,26 @@ _transcribe ()
 
 _typeset_functions ()
 {
+  # unfortunately, there does not seem to be a easy, portable way to list just the
+  # names of the defined shell functions...
+
+  case "$(_shell)" in
+    sh|bash)
+      typeset -F | awk '{print $3}'
+      ;;
+    *)
+      # trim everything following '()' in ksh/zsh
+      typeset +f | sed 's/().*$//'
+      ;;
+  esac
+}
+
+_typeset_functions_about ()
+{
   typeset f
-  for f in "$(_get_composure_dir)"/*.inc; do
-    # Without nullglob, we'll get back the glob
-    [[ -f "$f" ]] || continue
-
-    f="${f##*/}"
-    f="${f%.inc}"
-    echo "$f"
-  done | cat - <(echo "cite\ndraft\nglossary\nmetafor\nreference\nrevise\nwrite") | sort | uniq
+  for f in $(_typeset_functions); do
+    typeset -f "$f" | grep -qE "^about | about " && echo "$f"
```

Comment:
Why do you need that change? Because of "--" or is it the grep pattern?

Your version also works for me, but it is a bit slower as there are some more false positives, e. g, function references to "roundabout", which would be sorted out by my pattern.

(The time difference is only notable under cygwin as process forking is much, much slower there compared to 'real' linux)

can you track down why 

    grep -qE "^about | about "

isn't working?